src/spikeinterface/core/tests/test_whitematterrecordingextractor.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import numpy as np
+
+from spikeinterface.extractors import WhiteMatterRecordingExtractor, BinaryRecordingExtractor
+from spikeinterface.core.numpyextractors import NumpyRecording
+
+
+def test_round_trip(tmp_path):
```

Comment:
This is OK but I would prefer testing data. Can we add something to gin? This should be easy to stub.

Reply:
Ok, this is awaiting gin access. Should be up in a day or two.

Reply:
See https://gin.g-node.org/NeuralEnsemble/ephy_testing_data/pulls/155

Reply:
Ok, waiting for the test and should be good to go after that.

Reply:
> Ok, waiting for the test and should be good to go after that.

Done!